var/spack/repos/builtin/packages/warpx/package.py

```diff
@@ -38,14 +37,21 @@ class Warpx(MakefilePackage):
 
     resource(name='amrex',
              git='https://github.com/AMReX-Codes/amrex.git',
-             tag='development',
+             tag='master',
              destination='.')
 
     resource(name='picsar',
              git='https://bitbucket.org/berkeleylab/picsar.git',
              tag='master',
              destination='.')
 
+    @property
+    def build_targets(self):
+        if self.spec.satisfies('%clang'):
+            return ['CXXFLAGS=-std=c++11']
```

Comment:
better use 
```suggestion
            return ['CXXFLAGS={0}'.format(self.compiler.cxx11_flag)]
```

for broader compiler support, e.g. with older clang versions